dll/win32/shimgvw/lang/en-US.rc

```diff
@@ -2,19 +2,25 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 
 STRINGTABLE
 BEGIN
-    IDS_APPTITLE "ReactOS Picture and Fax Viewer"
-    IDS_SETASDESKBG "Set as Desktop Background"
-    IDS_NOPREVIEW "No preview available."
-    IDS_PREVIEW "Preview"
+    IDS_APPTITLE            "ReactOS Picture and Fax Viewer"
```

Comment:
Please revert stylistic changes, we usually don't align resource string tables, because this makes tracking (git blame) harder.

Reply:
I tried to remove that alignment in one file for doing an experiment, but it seems to me that it does not change too much because the files are still mostly different. Anyways, I will do it.